core/test/unit/services/webhooks_spec.js

```diff
@@ -6,6 +6,7 @@ var _ = require('lodash'),
 
     // Stuff we test
     webhooks = rewire('../../../server/services/webhooks'),
+    webhooksListen = rewire('../../../server/services/webhooks/listen'),
```

Comment:
Not sure here if this is the best way to rewire for packaged module functions like `listen` here 🤔